### ISSUE TITLE
Implement a learning rate warmup

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#36]
Note also that the ideal training hyperparameters seem to vary with model layer; I would imagine this extends to learning rate, and it certainly extends to the L1 regularization term.

**Comment by DavidUdell** [comment]
(Classified as "bug" because circuits aren't clean, presumably due to poor autoencoder solutions.)
